source/fundamentals/authentication/enterprise-mechanisms.txt

```diff
@@ -33,6 +33,14 @@ parameters of the
   parameter if using a value other than ``mongodb``
 - Specify a ``SERVICE_REALM`` value in the ``authMechanismProperties``
   parameter if a custom service realm is required.
+- Specify a ``CANONICALIZE_HOST_NAME`` value in the ``authMechanismProperties``
+  parameter if canonicalization of the hostname is required. This property can take
+  the following values:
+
+  - ``none``: (default) does not perform hostname canonicalization
+  - ``forward``: performs a forward DNS lookup to canonicalize the hostname
+  - ``forwardAndReverse``: performs a forward DNS lookup and then a
```

Comment:
S: Do these descriptions (the parts after the colons) look better if they start with a capital letter? Seems more natural to me.